index.d.ts

```diff
@@ -2,8 +2,9 @@ type ActionButton = {
   title?: string;
   description: string;
   iconPath: string;
-  action: any;
+  action?: any;
   disabled?: boolean;
+  url?: string;
```

Comment:
Not optional right now in the code as a <a> tag is rendered with the url always

Reply:
ActionButton is also used in Actions.tsx where the attribute `action` is used rather than `url` to open modals.